### ISSUE TITLE
EXPORT DATABASE on a table with current_timestamp as a default value does not work correctly

### ISSUE THREAD

**Issue by Mytherin** [duckdb/duckdb#1279]
Tables with the current_timestamp are not correctly exported because `current_timestamp` is seen as a function by the system, which is not accepted by the parser.

Example:

```sql
CREATE TABLE executions(id BIGINT PRIMARY KEY, time_start TIMESTAMP DEFAULT(current_timestamp), time_end TIMESTAMP, magic BIGINT NOT NULL DEFAULT(0));
EXPORT DATABASE 'test';
DROP TABLE executions;
IMPORT DATABASE 'test';
-- Error: Parser Error: syntax error at or near ")"
-- LINE 3: ...tart TIMESTAMP DEFAULT(current_timestamp()), time_end TIMESTAMP, magic BIGINT ...
```

**Comment by Mytherin** [comment]
CAST has the same problem.

**Comment by Mytherin** [comment]
Solved by #3098
